### PR TITLE
Update demo pipeline screenshot

### PR DESCRIPTION
**Pull request by MichaelKora**
Since the demo pipeline was updated, i just also updated the screenshot so it matches to the current pipeline.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/7.5.1/userguide/multi_project_builds.html
+ */
+
+rootProject.name = 'streams-explorer'
```
Comment (raminqaf) [inline settings.gradle:10]: The `streams-explorer` is not a Java project

File: gradlew.bat [hunk 2]
```diff
@@ -0,0 +1,91 @@
+@rem
```
Comment (raminqaf) [inline gradlew.bat:1]: Remove

**Review by raminqaf — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/*
```
Comment (yannick-roeder) [inline build.gradle:1]: We don't need Gradle in this project. Please remove this file and all other Gradle files

**Comment by disrupted** [comment]
thanks for the new screenshot! you can overwrite the previous `screens/demo-pipeline.png` directly

File: .gitignore [hunk 4]
```diff
@@ -21,3 +21,6 @@ helm-chart/index-cache.yaml
 .env*
 *private*
 mock/
+
+# Ignore Gradle build output directory
```
[the remark this answers is not in the document]
Comment (MichaelKora) [reply .gitignore:25]: you mean the comments?

File: gradlew [hunk 5]
```diff
@@ -0,0 +1,240 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (MichaelKora) [reply gradlew:1]: i actually have only changed the README file.

File: .gitattributes [hunk 6]
```diff
@@ -0,0 +1,9 @@
+#
```
Comment (disrupted) [inline .gitattributes:1]: what do we need this file for?

File: demo-atm-fraud/README.md [hunk 7]
```diff
@@ -33,21 +33,17 @@ helm upgrade --debug --install --force --values values-transactionjoiner.yaml de
...
 ### Generate test transactions
 
-To generate our incoming transactions (legitimate or fraudulent) we are using the `transactionavroproducer` application inspired by the 
-[gess](https://github.com/rmoff/gess) tool. This project's data is derived entirely from the same gess project.
-
+To generate our incoming transactions (legitimate or fraudulent) we are using the `transactionavroproducer` application inspired by the [gess](https://github.com/rmoff/gess) tool. This project's data is derived entirely from the same gess project.
 The number of incoming transactions is configurable using the variables `REAL_TX` and `ITERATION` in our `values-transactionavroproducer.yaml` file.
 The first variable specifies how many legitimate incoming transactions must be produced before producing one fraudulent incoming transaction.
 One iteration contains `REAL_TX` legitimate transactions and one fraudulent transaction.
-
```
Comment (disrupted) [inline demo-atm-fraud/README.md:53]: (unrelated to this PR) this is not necessary anymore thanks to your new transaction producer, right? If we don't need it anymore, we should remove or edit this section imo

Comment (disrupted) [reply demo-atm-fraud/README.md:53]: feel free to do it in a followup PR :)

File: .gitignore [hunk 8]
```diff
@@ -20,4 +20,4 @@ helm-chart/index-cache.yaml
 # local development files
 .env*
 *private*
-mock/
+mock/
```
Comment (raminqaf) [inline .gitignore:23]: newline :D

File: .gitattributes (hunk 6 above)
Comment (MichaelKora) [reply .gitattributes:1]: we dont need it at all. it was auto generated

File: demo-atm-fraud/README.md (hunk 7 above)
Comment (MichaelKora) [reply demo-atm-fraud/README.md:53]: will do

File: .gitignore (hunk 8 above)
Comment (yannick-roeder) [reply .gitignore:23]: This is not resolved yet, please add a new line at the end of the file

File: .gitattributes (hunk 6 above)
Comment (yannick-roeder) [reply .gitattributes:1]: This is not resolved yet, please remove this file from the MR

**Review by raminqaf — APPROVED** [review]

**Review by disrupted — APPROVED** [review]

**Review by yannick-roeder — APPROVED** [review]

**Pull request merged**